site/docs/reference/Connectors/materialization-connectors/starburst.md

```diff
@@ -0,0 +1,136 @@
+| **`/region`**             | AWS Region             | Region of AWS storage                                                                                              | string | Required         |
+| **`/bucket`**             | Bucket name            |                                                                                                                    | string | Required         |
+| **`/bucketPath`**         | Bucket path            | A prefix that will be used to store objects in S3.                                                                 | string | Required         |
+| /advanced                 | Advanced               | Options for advanced users. You should not typically need to modify these.                                         | string |                  |
+| /advanced/updateDelay     | Update Delay           | Potentially reduce active warehouse time by increasing the delay between updates. Defaults to 30 minutes if unset. | string | 30m              |
+
+#### Bindings
+
+| Property     | Title              | Description                       | Type   | Required/Default |
+|--------------|--------------------|-----------------------------------|--------|------------------|
+| **`/table`** | Table              | Table name                        | string | Required         |
+| `/schema`    | Alternative Schema | Alternative schema for this table | string |                  |
+
+### Sample
+
+```yaml
+
+materializations:
+  ${PREFIX}/${mat_name}:
+    endpoint:
+  	    connector:
+    	    config:
+              host: HOST:PORT
+              account: ACCOUNT
+              password: PASSWORD
+              catalog: CATALOG_NAME
+              schema: SCHEMA_NAME
+              awsAccessKeyId: AWS_ACCESS_KEY_ID
+              awsSecretAccessKey: AWS_SECRET_KEY_ID
+              region: REGION
+              bucket: BUCKET
+              bucketPath: BUCKET_PATH
+    	    image: ghcr.io/estuary/materialize-starburst:dev
+  # If you have multiple collections you need to materialize, add a binding for each one
+    # to ensure complete data flow-through
+    bindings:
+      - resource:
+          table: ${table_name}
+          schema: default
+    source: ${PREFIX}/${source_collection}
+```
+
+## Reserved words
+
+Galaxy has a list of reserved words that must be quoted in order to be used as an identifier. Flow automatically quotes fields that are in the reserved words list. You can find this list in Trino's documentation [here](https://trino.io/docs/current/language/reserved.html) and in the table below.
+
+:::caution
+In Galaxy, objects created with quoted identifiers must always be referenced exactly as created, including the quotes. Otherwise, SQL statements and queries can result in errors. See the [Trino docs](https://trino.io/docs/current/language/reserved.html#language-identifiers).
```

Comment:
quoted identifiers -> is rather abstract for a newbie in the Trino ecosystem.

Maybe provide an example.

Reply:
Example is in Trino docs.